src/Client.ts

```diff
@@ -344,12 +364,10 @@ export default class Client {
     return codec
   }
 
-  async encodeMessage(
-    recipient: PublicKeyBundle,
-    timestamp: Date,
+  async encodeContent(
```

Comment:
extracting the content encoding bits to standalone methods so that they can be reused for V2.